src/components/DocsNavs.re

```diff
@@ -762,18 +762,18 @@ module Dropdown = {
           title={intl->Intl.formatMessage(howZkAppUI)}
           slug="how-to-write-a-zkapp-ui"
           />
-          <Item
-            title={intl->Intl.formatMessage(zkAppsForEthereumDevelopers)}
-            slug="zkapps-for-ethereum-developers"
-          />
-          <Item
-            title={intl->Intl.formatMessage(simpleAnonymousMessageBoardTutorial)}
-            slug="simple-anonymous-message-board-tutorial"
-          />
           <Item
           title={intl->Intl.formatMessage(advancedConcepts)}
           slug="advanced-concepts"
```

Comment:
Should be deleted? I don't see 2x on the UI, so not sure

Reply:
Not deleted but changed to snarkyjs